web_monitoring/diffing_server.py

```diff
@@ -90,7 +102,7 @@ def get(self, differ):
                 actual_hash = hashlib.sha256(res.body).hexdigest()
                 if actual_hash != expected_hash:
                     self.send_error(
-                        500, reason="Fetched content does not match hash.")
+                        500, reason = 'Fetched content does not match hash.')
```

Comment:
Elaborating on @Mr0grog's request for spaces around operators, Python's recommended style does not put spaces around `=` when it's used in a keyword argument like

```py
def f(x=3):
    ...

f(x=2)
```

only when it's used in a assignment like

```py
y = 6
```

You can use a tool like [flake8](http://flake8.pycqa.org/en/latest/) to check the code style and catch these things.

Reply:
Thank you @danielballan ! Having worked for 6 months mostly with Swift, the tool you suggested is a great way to remember Python's coding style.